post_processing/contours_and_profiles.jl

```diff
@@ -69,15 +69,25 @@ function contour_plot!(
     end
     ts = getindex.(var_col_time_series, 1)
     zs = var_col_time_series[1][2]
+
+    #values_adjoint = hcat(getindex.(var_col_time_series, 3)...)
```

Comment:
remove this?

Reply:
Could we keep them for now? The plotting as it currently is done on main doesn't work on my local machine. The two commented out lines is what is working on my local. (Maybe someone else will run into this problem and the thing to try then would be to replace the line below with the commented out lines)

Reply:
Ah, ok, sounds good!